packages/node/src/handlers.ts

```diff
@@ -200,7 +201,20 @@ export function errorHandler(): (
 /** JSDoc */
 export function defaultOnFatalError(error: Error): void {
   console.error(error && error.stack ? error.stack : error);
-  global.process.exit(1);
+  const options = (getCurrentHub().getClient() as NodeClient).getOptions();
+  const timeout =
+    (options && options.shutdownTimeout && options.shutdownTimeout > 0 && options.shutdownTimeout) || 2000;
+  (getCurrentHub().getClient() as NodeClient)
+    .close(timeout)
+    .then((result: boolean) => {
+      if (!result) {
+        logger.warn('We reached the timeout for emptying the request buffer, still exiting now!');
+      }
+      global.process.exit(1);
+    })
+    .catch(() => {
```

Comment:
We have a `forget` util which can be used here

Reply:
Woohoo 🎉